lib/hawkular/inventory/inventory_api.rb

```diff
@@ -38,39 +38,32 @@ def self.create(hash)
       Client.new(hash[:entrypoint], hash[:credentials], hash[:options])
     end
 
-    # Retrieve the tenant id for the passed credentials.
-    # If no credentials are passed, the ones from the constructor are used
-    # @param credentials [Hash{String=>String}] Hash of username, password, token(optional)
-    # @return [String] tenant id
-    # @deprecated this doesn't provide any value, because it merely returns the tenant ID which is known before the
-    # call anyway.
-    def get_tenant(credentials = {})
-      creds = credentials.empty? ? @credentials : credentials
-      auth_header = { Authorization: base_64_credentials(creds) }
-
-      ret = http_get('/tenant', auth_header)
-
-      ret['id']
-    end
-
     # List feeds in the system
     # @return [Array<String>] List of feed ids
     def list_feeds
-      ret = http_get('/traversal/type=f')
-      ret.map { |f| f['id'] }
+      ret = http_get('/strings/tags/module:inventory,feed:*')
+      ret['feed']
     end
 
     # List resource types. If no feed_id is given all types are listed
     # @param [String] feed_id The id of the feed the type lives under. Can be nil for feedless types
     # @return [Array<ResourceType>] List of types, that can be empty
-    def list_resource_types(feed_id = nil)
-      if feed_id.nil?
-        ret = http_get('/traversal/type=rt')
-      else
-        the_feed = hawk_escape_id feed_id
-        ret = http_get("/traversal/f;#{the_feed}/type=rt")
+    def list_resource_types(feed_id)
```

Comment:
The feed_id was not mandatory in the previous API, but now it is. I checked in MIQ, there should always be a feed id.